src/main/java/ddingdong/ddingdongBE/domain/filemetadata/service/FileMetaDataServiceImpl.java

```diff
@@ -40,11 +39,13 @@ public List<FileMetaData> getCoupledAllByDomainTypeAndEntityId(DomainType domain
     @Transactional
     @Override
     public void updateStatusToCoupled(List<String> ids, DomainType domainType, Long entityId) {
-        List<UUID> fileMetaDataId = toUUIDs(Objects.requireNonNullElse(ids, Collections.emptyList()));
+        if (ids.isEmpty()) {
+            return;
+        }
+        List<UUID> fileMetaDataId = toUUIDs(ids);
         List<FileMetaData> fileMetaDatas = fileMetaDataRepository.findByIdIn(fileMetaDataId);
         if (fileMetaDatas.isEmpty()) {
-            log.warn("fileMetaData를 찾을 수 없습니다. requestIds={}", ids);
-            return;
+            throw new ResourceNotFound("해당 FileMetaData(id: " + fileMetaDataId + ")를 찾을 수 없습니다.");
```

Comment:
p3)
요건 in절로 쿼리를 날리기 때문에 ids의 개수와 다르게 조회될 경우도 에러로 처리해야하기 때문에 다음과 같이 수정하는건 어떨까요?
```suggestion
	if (ids.size() != fileMetaDatas.size()) {
            throw new ResourceNotFound("해당 FileMetaData(ids: " + ids + ")를 찾을 수 없습니다.");
```

Reply:
확인하였습니다